### PR TITLE
android notification docs

### PR DESCRIPTION
**Pull request by KennyHuRadar**
## What?

Documentation on how to use the new android notification settings
## Why?

There were SDK changes made per request of burgerville. 
## How?

An android specific function `setNotificationOptions` was added
## Screenshots (optional)

RN
image
Android
image

### CODE REVIEW

File: docs/sdk/android.mdx [hunk 1]
```diff
@@ -1533,6 +1533,71 @@ With the [conversions API](/api#log-a-conversion), log a conversion, such as a p
...
+    { label: 'Kotlin', value: 'kotlin' }
+  ]}
+>
+  <TabItem value="java">
+
+  ```java
+  JSONObject notificationOptions = new JSONObject();
+  try {
+    notificationOptions.put("iconString","icon_1");
+    notificationOptions.put("iconColor","color_1");
```
Comment (jsani-radar) [inline docs/sdk/android.mdx:1553]: I'm assuming this works across fg service / event notifications? not 100% clear from this snippet

File: docs/sdk/android.mdx [hunk 2]
```diff
@@ -1533,6 +1533,71 @@ With the [conversions API](/api#log-a-conversion), log a conversion, such as a p
...
+  } catch (JSONException e) {
+    e.printStackTrace();
```
Comment (jsani-radar) [inline docs/sdk/android.mdx:1555]: wondering why there'd ever be an exception here? it's just adding a k-v pair to a JSON dict

File: docs/sdk/android.mdx [hunk 3]
```diff
@@ -1533,6 +1533,71 @@ With the [conversions API](/api#log-a-conversion), log a conversion, such as a p
...
+  ```
+
+  </TabItem>
+  <TabItem value="kotlin">
+
+  ```kotlin
+  Radar.setNotificationsOptions(RadarNotificationsOptions(
+    "icon_1",
+    "color_1",
+  ))
```
Comment (jsani-radar) [inline docs/sdk/android.mdx:1584]: I'm assuming these need the `iconString` / `iconColor` keys like we have with Java above?

File: docs/sdk/react-native.mdx [hunk 4]
```diff
@@ -621,6 +621,26 @@ Radar.logConversion(
   // optionally, do something with err
 });
 ```
+
+### Notifications (Android only)
```
Comment (jsani-radar) [inline docs/sdk/react-native.mdx:625]: are they really Android only? I could see a world where we support custom icons / colors for event notifications on iOS

in fact, we might want to for parity / consistency

**Review by jsani-radar — COMMENTED** [review]
looking good overall!

few small comments - perhaps from my lack of knowledge around this new fn

File: docs/sdk/android.mdx [hunk 5]
```diff
@@ -1533,6 +1533,71 @@ With the [conversions API](/api#log-a-conversion), log a conversion, such as a p
...
+  ```java
+  JSONObject notificationOptions = new JSONObject();
+  try {
+    notificationOptions.put("iconString","icon_1");
+    notificationOptions.put("iconColor","color_1");
+  } catch (JSONException e) {
+    e.printStackTrace();
+  }
+
+  Radar.setNotificationsOptions(RadarNotificationsOptions.fromJson(notificationOptions));
```
Comment (jsani-radar) [inline docs/sdk/android.mdx:1558]: is it `setNotificationOptions` or `setNotificationsOptions` (singular or plural)? seeing a mix of the two

File: docs/sdk/react-native.mdx (hunk 4 above)
Comment (lmeier) [reply docs/sdk/react-native.mdx]: they are android only for now — we can figure out iOS when the need arises

**Review by lmeier — APPROVED** [review]

**Review by jsani-radar — APPROVED** [review]

**Pull request merged**